docs/specifications/tests/Nameserver-TP/nameserver14.md

```diff
@@ -0,0 +1,87 @@
+# NAMESERVER14: Test for unknown version with unknown OPTION-CODE
+
+## Test case identifier
+
+**NAMESERVER14** 
+
+### Objective
+
+This test case actually combines the test options in test cases
+[NAMESERVER10](../nameserver10.md) and [NAMESERVER11](../nameserver11.md). 
+
+### Inputs
+
+"Child Zone" - The domain name to be tested.
+
+### Ordered description of steps to be taken to execute the test case
+
+1. Create an SOA query for the *Child Zone* with an OPT record with 
+   EDNS version set to "1" and  with EDNS OPTION-CODE set to an
+   unassigned value (see [IANA-DNSSYSTEM-PARAMETERS]) and
+   with no other EDNS options or flags. 
+
+2. Obtain the set of name server IP addresses using [Method4] and [Method5]
+   ("Name Server IP").
+
+3. For each name server in *Name Server IP* do:
+
+	1. Send the SOA query to the name server and collect the response.
+	2. If there is no DNS response, output *[NO_RESPONSE]* and go to
+      	next server.
+	3. Else, if the DNS response has the RCODE "FORMERR" then output
+      	*[NO_EDNS_SUPPORT]*.
+	4. Else, if the RCODE has "NOERROR", and has an "OPTION-CODE" and
+	"Version Number" present in the response,  then output
+	[UNKNOWN_OPTION_CODE_VERSION]. 
```

Comment:
I suggest:

4. Else, if the RCODE has "NOERROR", and has an OPT record with "OPTION-CODE", then output
    [UNKNOWN_OPTION_CODE], and if the OPT version is greater than zero, then also output
    [UNSUPPORTED_EDNS_VER].

Reply:
@sandoche2k, you did not like my proposal?

```
	4. Else, if the RCODE has "NOERROR", and has an "OPTION-CODE" and
            (...)
```
If the response has an OPTION-CODE set it means that an OPT record is present. For the clarity I think we should say that.
```
        (...)
	"Version Number" present in the response,  then output
        (...)
```
If there is an OPT record there is, by definition, the EDNS version is always set. That is nothing to test for.
```
        (...)
	[UNKNOWN_OPTION_CODE_VERSION]. 
        (...)
```
What would this combination message say? What not use the two different messages instead?